geopandas/plotting.py

```diff
@@ -40,6 +40,45 @@ def _flatten_multi_geoms(geoms, prefix="Multi"):
     return components, np.array(component_index)
 
 
+def _expand_kwargs(kwargs, multiindex):
```

Comment:
I like the way this is handled. It is clear and expandable if we find other kwargs which needs to be handled individually.